src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Please make your App.js component class-based, this is part of the project requirements 👍🏿

Reply:
Hello @prolajumokeoni could you please check again or clarify, because I believe I have a components folder that has class-based components in the Calculator.js  just as the project requirements.

Thank you